assets/config/0.5.5.1-coins.json

```diff
@@ -10338,5 +10338,55 @@
     "active": false,
     "is_segwit_on": false,
     "currently_enabled": false
+  },
+"ACTN": {
+    "coin": "ACTN",
+    "name": "Action Coin",
+    "type": "Smart Chain",
+    "coinpaprika_id": "actn-action-coin",
+    "coingecko_id": "action-coin",
+    "nomics_id": "actn-action-coin",
+    "electrum": [
+      {
+        "url": "electrumx1.actioncoin.com:30001",
+        "ws_url": "electrumx1.actioncoin.com:20001",
+        "protocol": "SSL"
+      },
+      {
+        "url": "electrumx2.actioncoin.com:30001",
+        "ws_url": "electrumx2.actioncoin.com:20001",
+        "protocol": "SSL"
+      },	
+      {
+        "url": "electrumx1.actioncoin.com:10001"
```

Comment:
there is a missing `,`after this line
you could also remove the next line completely, since TCP is the default

Reply:
Done